### DISCUSSION TITLE
Clickable cards bug resolution

### DISCUSSION THREAD

**Discussion by evwilkin** [patternfly/patternfly#6381] [Announcements]
Hello PatternFly community, we wanted to share an update & resolution to a bug that affected PatternFly this week with regards to a recent Chromium update - please note that the fix has been published with our latest patch release `5.2.1` with full release notes linked below.  

*Thanks to our community for bringing this to our attention and to the PatternFly team for acting quickly to triage, understand, and resolve the problem - please don’t hesitate to contact us with any questions, concerns, or suggestions.*

---

**What happened?**
- Several Red Hat product teams have reported an issue with our PatternFly cards click event. We have traced it to a recent Chromium update.

**What was the Chromium update?**
- The Chromium update is a progression in how Chrome now supports align-self/justify-self on fixed and absolutely positioned elements. See the [Chromium closed/won't fix bug](https://issues.chromium.org/issues/326980470) for reference.

**How we fixed this**
1. As a team, we reviewed the part of the specification that was implemented and resulted in the reported bug.  Specifically that *-self alignment properties now apply to absolutely/fixed positioned elements. (https://www.w3.org/TR/css-align-3/#overview)
2. Then we discussed possible solutions. Out of several possible fixes, we determined that it was safest to simply revert the self alignment properties if they apply to an absolutely/fixed positioned element, because it would not need them for alignment.
3. We identified every selector across all PatternFly stylesheets that use a self alignment property, and manually reviewed everywhere any of the selectors are used to ensure that we are not combining a self alignment property with absolute or fixed positioning.
4. Finally, we ran visual regression testing before and after the patch fix to ensure that the regression testing picked up on the reported card bug and that no other instances were found.

**Plans to avoid this in the future**
- For the future, we plan to extend the visual regression tests to run against more browsers so that issues like this should be picked up more quickly.

**Additional details:**
- Core patch release notes: https://github.com/patternfly/patternfly/releases/tag/patch-v5.2.1
- React patch release notes: https://github.com/patternfly/patternfly-react/releases/tag/5.2.1

**Comment by ezr-ondrej** [comment]
Hi I've bumped to 5.2.1, but clickable cards still do not work on chrome 122

[Screencast from 2024-03-05 15-31-03.webm](https://github.com/patternfly/patternfly/assets/2884324/c89b8a46-8fee-48c6-a96a-3c4a85db6fc0)

**Reply by wise-king-sullyman** [reply]
Can you share the repo in question?
